### ISSUE TITLE
Failed to reproduce results

### ISSUE THREAD

**Issue by yuyu2172** [meteorshowers/RCF-pytorch#5]
I failed to reproduce the results on the README using your training script.
The scores were

```
ods: 0.7926  (README says I should get 0.808)
ois: 0.8073   (README says I should get 0.811)
```

The trained weights can be found here https://drive.google.com/file/d/10N9ohhzJvHdbrYfVd_1zzJ-vp46XEiZN/view?usp=sharing.
The code I used is here https://github.com/yuyu2172/RCF-pytorch.

If possible, could you upload the weights of the trained model you used to produce the results on README?
Also, could you upload VGG16 pretrained weights?

**Comment by meteorshowers** [comment]
your result is common.  The pytorch version's convergence speed is  faster than the caffe version's.
I get the best result in the training stage, not the last stage, however, I forgert the detail now.
I am doing my work for ddl now. So, I will consider update the readme in detail after my ddl.
Have a good luck~

**Comment by 22920132203749** [comment]
@yuyu2172 where did you got the vgg16convs.mat file？

**Comment by mayorx** [comment]
Failed to reproduce results.......
I get ods: 0.76 on your @yuyu2172 trained model and 0.75 on mine. (BSDS500, test set)

I use [Structured Edge Detection Toolbox](https://github.com/pdollar/edges) to evaluate my results, but I'm not sure of the parameters of [edgesEvalDir.m](https://github.com/pdollar/edges/blob/master/edgesEvalDir.m), could you please tell me yours? @meteorshowers @yuyu2172 

Here is my parameters:
thrs: 99
maxDist: 0.0075
pDistr: {'type'  'parfor'}
thin: 1

**Comment by mayorx** [comment]
### Here are the results by using @yuyu2172 's model.

![](https://i.postimg.cc/6q2N7fCk/2019-02-08-7-49-32.png)

### Here are the results of mine.
![](https://i.postimg.cc/VNgw26YV/2019-02-08-7-49-45.png)

**Comment by mayorx** [comment]
> ### Here are the results by using @yuyu2172 's model.

> .....

Because it need nms first ... 
before nms:
<img src="https://github.com/mayorx/rcf/blob/master/examples/100007-6.png?raw=true" width="200" />
after nms:
<img src="https://github.com/mayorx/rcf/blob/master/examples/100007-nms.png?raw=true" width="200">
[Here](https://github.com/mayorx/rcf#evaluate) is an example of the evaluation process.

**Comment by MeLonJ10** [comment]
> I failed to reproduce the results on the README using your training script.
> The scores were
> 
> ```
> ods: 0.7926  (README says I should get 0.808)
> ois: 0.8073   (README says I should get 0.811)
> ```
> 
> The trained weights can be found here https://drive.google.com/file/d/10N9ohhzJvHdbrYfVd_1zzJ-vp46XEiZN/view?usp=sharing.
> The code I used is here https://github.com/yuyu2172/RCF-pytorch.
> 
> If possible, could you upload the weights of the trained model you used to produce the results on README?
> Also, could you upload VGG16 pretrained weights?

Hi~ Have u done any modification, such as learning rate, the type of optimizer, or data augmentation? Because I got wrong results where the last two ouput is totally black.
Thanks so much for ur kind help.

**Comment by cainiaoqiuzhu** [comment]
> ### Here are the results by using @yuyu2172 's model.
> ![](https://camo.githubusercontent.com/83dafd7777028f6bb4dcfa71ffbe159433364f86d1b1ae51fff20000af519e84/68747470733a2f2f692e706f7374696d672e63632f3671324e3766436b2f323031392d30322d30382d372d34392d33322e706e67)
> 
> ### Here are the results of mine.
> ![](https://camo.githubusercontent.com/86b3e0e8d673a26eaa4df97e8ff688a244bbe762de7dcceee4e8de4f73224f77/68747470733a2f2f692e706f7374696d672e63632f564e6777323659562f323031392d30322d30382d372d34392d34352e706e67)

![image](https://user-images.githubusercontent.com/24868875/113238775-00dee480-92dc-11eb-8a59-84f1e02d1c85.png)
this is my result.
